### ISSUE TITLE
on_left_side property

### ISSUE THREAD

**Issue by pberkes** [ASPP/pelita#49]
property that returns True when the agent starts on left side of maze:
useful for example in:

```
    forbidden = west if self.on_left_side else east
    if self.maze_nodes[pos] in self.border_nodes and action == forbidden:
        return stop
```

**Comment by sappelhoff** [comment]
I have some trouble adding a test.

When adding  the following to `test/test_datamodel.py`:

    def test_on_west_side(self):
        bot = Bot(1, (6, 6), 1, (3, 6), current_pos = (1, 1))
        assert bot.on_west_side

I get this error upon running `pytest test/test_datamodel.py`:

`E       AttributeError: 'Bot' object has no attribute 'on_west_side'`

Furthermore, when I go to `datamodel.py` and change e.g., the `is_harvester` method to `is_harvester2`, the tests still pass despite the fact that the `is_harvester` method should be missing ...

Any idea why this happens @Debilski?

**Comment by Debilski** [comment]
@sappelhoff Answering in #429

**Comment by Debilski** [comment]
Closed in #430
